### PR TITLE
3 manual continued

### PR DESCRIPTION
**Pull request by stardusthamster**
Just some small changes concerning the shutdown and added another todo...

### CODE REVIEW

File: docs/manual/manual.md [hunk 1]
```diff
@@ -556,6 +563,15 @@ TODO Work in Progress....
...
 
+<a name="#howto-tigger-action"></a>
+
+#### Howto trigger an action using fingering
+
+TODO Work in Progress....
+
+Some time ago there existed a [key combination to trigger the shutdown](https://github.com/cardonabits/haxo-rs/commit/9a9a84b224fc8a8fe0d9163508b1aee2d814d735).
+
+
```
Comment (jcard0na) [inline docs/manual/manual.md:574]: I would not introduce a new term (`actions`) given that we already have `commands`.  Also, I would not title a section `How to do THIS` just to say in the body that `THIS` is not possible...  🤔😁)

```suggestion
<a name="#howto-tigger-shutdown"></a>

#### The non-existent Shutdown command

Some time ago there existed a [shutdown command](https://github.com/cardonabits/haxo-rs/commit/9a9a84b224fc8a8fe0d9163508b1aee2d814d735) but it was removed as it was easy to trigger accidentally while playing.
Now that we have more robust method to enter Command Mode that is much harder to trigger accidentally, maybe we should bring back the shutdown command.  If you want it, feel free to open an issue, or even better, a Pull Request.
```

File: docs/manual/manual.md [hunk 2]
```diff
@@ -228,7 +230,12 @@ Now your haxophone is ready to play!
    Blow air through the mouthpiece or tip of the neck to begin playing.
 
 4. **Stop Playing:**
```
Comment (jcard0na) [inline docs/manual/manual.md:232]: With the new section you are adding right below this, this section should probably be removed.
```suggestion
```

File: docs/manual/manual.md [hunk 3]
```diff
@@ -228,7 +230,12 @@ Now your haxophone is ready to play!
...
-When you have finished playing and want to power off the haxophone, just remove the USB power. As the SD card is used in readonly mode, there is no danger in corrupting the data on the SD card when removing power.
+When you have finished playing and want to power off the haxophone, just remove the USB power. As the SD card is used in read only mode, there is no danger in corrupting the data on the SD card when removing power.
+
+ <a name="stop-haxophone"></a>
+
+#### Shutdown the Haxophone
+To turn off the Haxophone you just need to unplug the power. You don't need to worry about corrupting files on the SD card, as the SD card is read only per default.
```
Comment (jcard0na) [inline docs/manual/manual.md:238]:
```suggestion
 <a name="stop-haxophone"></a>

#### Shutdown the Haxophone
To turn off the Haxophone you just need to unplug the power. You don't need to worry about corrupting files on the SD card, as the SD card is read only per default.
```

File: docs/manual/manual.md [hunk 4]
```diff
@@ -24,6 +25,7 @@
 		+ [Install the Newest Binary](#install-newest-binary)
 		+ [Howto Change the Sound Set](#howto-change-sound-set)
 		+ [Howto Change the notemap.json](#howto-notemap.json)
+		+ [Howto trigger an action using fingering](#howto-tigger-action)
```
Comment (jcard0na) [inline docs/manual/manual.md:28]:
```suggestion
		+ [The non-existent Shutdown command](#howto-tigger-shutdown)
```

File: docs/manual/manual.md [hunk 5]
```diff
@@ -755,4 +773,4 @@ For an explanation how to switch between the sounds see
 
 | Document Version | Changes |
 | --- | --- |
-| 0.2 |     |
+| 0.3 | Added howto shutdown  |
```
Comment (jcard0na) [inline docs/manual/manual.md:776]: Normally this should be a version table showing all versions, not just the last change...

**Review by jcard0na — APPROVED** [review]
Made some suggestions, but feel free to cherry pick or ignore at will...

**Comment by stardusthamster** [comment]
Thank you for the very detailed feedback!

**Pull request merged**